src/components/sykmeldingpanel/AnnenInfo.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { Information } from '@navikt/ds-icons'
+
+import { IconHeading } from '../shared/IconHeading/IconHeading'
+import { SykmeldingFragment } from '../../graphql/queries/graphql.generated'
+import { formatDate } from '../../utils/dateUtils'
+import { cleanId } from '../../utils/stringUtils'
+import { isUtenlandsk } from '../../utils/utenlanskUtils'
+import { ListItem } from '../shared/listItem/ListItem'
+
+import styles from './AnnenInfo.module.css'
+
+interface Props {
+    sykmelding: SykmeldingFragment
+}
+
+function AnnenInfo({ sykmelding }: Props): JSX.Element {
+    const title = 'Annen info'
+    return (
+        <li aria-labelledby={cleanId(title)}>
+            <IconHeading title={title} Icon={Information} />
```

Comment:
Om jeg forstår det rett så peker denne labelledby-en på en id inne i `IconHeading`? Det virker litt løst koblet, og enkelt å bryte. Kanskje det gir mer mening å sende inn id eksplisitt?

```suggestion
        <li aria-labelledby={cleanId(title)}>
            <IconHeading headingId={cleanId(title)} title={title} Icon={Information} />
```

selv om det strengt talt er overkill?